controllers/build/create_binaries.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -x
+set -o errexit
+set -o pipefail
+
+GOLANG_VERSION="${1?Specify fourth argument - golang version}"
+BINARY_NAME="${2?Specify fifth argument - binary name}"
+IMAGE_REPO="${3?Specify sixth argument - ecr image repo}"
+IMAGE_TAG="${4?Specify seventh argument - ecr image tag}"
+
+BIN_ROOT="_output/bin"
+BIN_PATH=$BIN_ROOT/$BINARY_NAME
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+source "$REPO_ROOT/scripts/common.sh"
+
+KUSTOMIZE_BIN="${REPO_ROOT}/_output/kustomize-bin"
+
+function build::install::kustomize(){
+  curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash
+  mv kustomize $KUSTOMIZE_BIN
+  export PATH=$KUSTOMIZE_BIN:$PATH
+}
+
+function build::eks-anywhere-cluster-controller::create_binaries(){
+  platform=${1}
+  OS="$(cut -d '/' -f1 <<< ${platform})"
+  ARCH="$(cut -d '/' -f2 <<< ${platform})"
+  CGO_ENABLED=0 GOOS=$OS GOARCH=$ARCH make build-cluster-controller-binaries
+  mkdir -p ../${BIN_PATH}/${OS}-${ARCH}/
```

Comment:
In the build tooling repo, we ran Go builds after `cd`ing into the upstream project we cloned, and copied the binary one level up, which is where the project Makefile and `_output` folder was. Here, we dont clone anything, and we call Make from the root, which is where `bin`, `_output` and Makefile are, i.e., everything is at the same level.